modules/database/src/ioc/db/dbChannel.h

```diff
@@ -54,6 +55,8 @@ typedef struct chFilter chFilter;
 typedef struct dbChannel {
     const char *name;
     dbAddr addr;              /* address structure for record/field */
+    dbAddrModifier addrModifier;
+                              /* optional: which indices are targeted */
```

Comment:
Could `dbHandleAddrModifier` be embedding directly in `dbChannel`?  Seems like this could avoid the need for a new public `dbPutModifier()` and similar.

Reply:
Again, see my general comment about the extension point.